app/controllers/users.js

```diff
@@ -106,3 +101,25 @@ exports.list = (req, res, next) => {
     next(err);
   });
 };
+
+exports.signout = (req, res, next) => {
+  userService.signout(req.user).then((signedOutUser) => {
+    res.status(200);
+    res.end();
+  }).catch((err) => {
+    next(err);
+  });
+};
+
+exports.createAdmin = (req, res, next) => {
+  const newUser = req.body;
+  emailValidation(newUser.email);
+  passwordValidation(newUser.password);
+  userValidation(newUser);
+  userService.createOrUpdateAdmin(newUser, { isAdmin: true }).then((user) => {
```

Comment:
If the function is called `createOrUpdateAdmin` why does it need to receive the `isAdmin: true` data? I think maybe the name isn't the most descriptive for the actual functionality.